packages/mapviewer/src/modules/map/components/openlayers/utils/usePrintAreaRenderer.composable.js

```diff
@@ -1,14 +1,21 @@
 import log from '@geoadmin/log'
 import * as olHas from 'ol/has'
+import VectorLayer from 'ol/layer/Vector'
 import { getRenderPixel } from 'ol/render'
+import VectorSource from 'ol/source/Vector'
 import { computed, watch } from 'vue'
 import { useStore } from 'vuex'
 
 const dispatcher = { dispatcher: 'print-area-renderer.composable' }
-
+function createSourceForProjection() {
+    return new VectorSource({
+        useSpatialIndex: false,
+        wrapX: true,
+    })
+}
```

Comment:
I don't think you are creating this specifically for a projection.
And it looks like you only use it once, so it can simply be declared inline with the `new VectorLayer` below, IMHO